lib/ts/recipe/oauth2provider/types.ts

```diff
@@ -358,6 +358,13 @@ export type RecipeInterface = {
         tenantId: string;
         userContext: UserContext;
     }): Promise<JSONObject>;
+    revokeToken(input: {
+        token: string;
+        clientId?: string;
+        clientSecret?: string;
+        authorizationHeader?: string;
```

Comment:
This could become a "split type" to signify that we either get id+secret or a header. If it introduces inconveniences elsewhere we don't need to do it, but please consider this.

Reply:
done